Vite example doesn't install

Installation of the Vite example fails with unresolvable dependencies.
Also, the code that puts the app on the page is commented out.

```
❯ npm i
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: vite-plugin-solid@2.7.2
npm ERR! Found: vite@5.0.13
npm ERR! node_modules/vite
npm ERR!   dev vite@"^5.0.13" from the root project
npm ERR!   peerOptional vite@"^3.0.0 || ^4.0.0 || ^5.0.0" from vitefu@0.2.5
npm ERR!   node_modules/vitefu
npm ERR!     vitefu@"^0.2.4" from vite-plugin-solid@2.7.2
npm ERR!     node_modules/vite-plugin-solid
npm ERR!       dev vite-plugin-solid@"^2.7.2" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer vite@"^3.0.0 || ^4.0.0" from vite-plugin-solid@2.7.2
npm ERR! node_modules/vite-plugin-solid
npm ERR!   dev vite-plugin-solid@"^2.7.2" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: vite@4.5.3
npm ERR! node_modules/vite
npm ERR!   peer vite@"^3.0.0 || ^4.0.0" from vite-plugin-solid@2.7.2
npm ERR!   node_modules/vite-plugin-solid
npm ERR!     dev vite-plugin-solid@"^2.7.2" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

--- a/examples/vite/package.json
+++ b/examples/vite/package.json
@@ -16,6 +16,6 @@
     "@types/workerpool": "^6.4.7",
     "typescript": "^5.3.2",
     "vite": "^5.0.13",
-    "vite-plugin-solid": "^2.7.2"
+    "vite-plugin-solid": "^2.10.2"
   }
 }

--- a/examples/vite/src/index.tsx
+++ b/examples/vite/src/index.tsx
@@ -1,7 +1,7 @@
 /* @refresh reload */
-// import { render } from 'solid-js/web'
-// import App from './App'
+import { render } from 'solid-js/web'
+import App from './App'
 
-// const root = document.getElementById('root')
+const root = document.getElementById('root')
 
-// render(() => <App />, root!)
+render(() => <App />, root!)

